### PR TITLE
infra setup

### PR DESCRIPTION
**Pull request by ramki11**

### CODE REVIEW

**Comment by ramki11** [comment]
this PR is for setting up core infra

File: infra/BillingAlert.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+resource "aws_sns_topic" "billing_alert_topic" {
+  name = "BillingAlertTopic"
+}
+
+resource "aws_sns_topic_subscription" "email_subscription" {
+  topic_arn = aws_sns_topic.billing_alert_topic.arn
+  protocol  = "email"
+  endpoint  = "ramki.ratnakara@gmail.com" # Replace with your email address
```
Comment (Eugene-Polonsky) [inline infra/BillingAlert.tf:28]: this should probably go to me -- eugene@incredihire.com

File: .github/workflows/infra-setup.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v3
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
```
Comment (Eugene-Polonsky) [inline .github/workflows/infra-setup.yml:18]: should this be west?

Comment (ramki11) [reply .github/workflows/infra-setup.yml:18]: updated with variable

**Comment by ramki11** [comment]
Drop this PR as this is using the fork.

**Pull request closed without merging**